openml/runs/run.py

```diff
@@ -442,7 +442,9 @@ def extract_parameters(_flow, _flow_dict, component_model,
                         # inside a feature union or pipeline
                         if not isinstance(_tmp, (list, tuple)):
                             raise e
-                        for step_name, step in _tmp:
+                        for _temp_step in _tmp:
```

Comment:
Maybe we should also add some checks/safeguards that the third argument is not an instance of `OpenMLFlow`? Meybe this piece of code can be generalized to check all elements in that list?

Reply:
I added checks for all 3 parts of the tuple